pkg/cluster/kubeclient/backup.go

```diff
@@ -13,9 +13,9 @@ import (
 	"github.com/openshift/openshift-azure/pkg/util/wait"
 )
 
-func (u *kubeclient) BackupCluster(ctx context.Context, backupName string) error {
-	u.log.Infof("running an etcd backup")
-	cronjob, err := u.client.BatchV1beta1().CronJobs("openshift-etcd").Get("etcd-backup", metav1.GetOptions{})
+func (u *KubeclientT) BackupCluster(ctx context.Context, backupName string) error {
```

Comment:
change everywhere

```suggestion
func (u *Kubeclient) BackupCluster(ctx context.Context, backupName string) error {
```

Reply:
the interface name is Kubeclient, so I made the struct name KubeclientT.
I am not crazy about it.. I could change the Kubeclient->Interface and kubeclient->Kubeclient?
but if i do just as you suggest there will be a name clash..

Reply:
fair enough :)